ioctl/cmd/action/actiondeploy.go

```diff
@@ -1,63 +0,0 @@
-// Copyright (c) 2019 IoTeX Foundation
```

Comment:
contract deploy/invoke cmd are still WIP, for now keep these 2 files
we can revisit and make the decision to delete or not, once the contract cmds are done

Reply:
Ok, I will make an issue for it then